functions-and-scope.js

```diff
@@ -41,7 +76,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6.642857142857143
-
+//Declareer variable sumGrades om de uitkomst in op te slaan
+//Maak een loop die door de array loopt
+//Bij elke loop sla het cijfer op in de sumGrades door deze erbij op te tellen
+//Bereken het gemiddelde door de formule sumGrades / array.length toe te passen
+//Log de uitkomst in de console
+let sumGrades = 0;
+for(i=0; i < grades.length; i++) {
+    sumGrades = sumGrades + grades[i];
+}
```

Comment:
In principe zou je hiervoor de 'reduce' functie kunnen gebruiken:
```
const sumGrades = grades.reduce((total, grade) => total + grade);
```

Ter info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce